ci/requirements-2.7.sh

```diff
@@ -4,4 +4,4 @@ source activate pandas
 
 echo "install 27"
 
-conda install -n pandas -c conda-forge feather-format pyarrow=0.4.1 fastparquet
+conda install -n pandas -c conda-forge feather-format pyarrow=0.7.0 fastparquet
```

Comment:
let's keep the old version here? (so we actually test the older version, now we dropped bumping the version in this PR)